.vitepress/config.mts

```diff
@@ -35,6 +35,339 @@ function nav(): DefaultTheme.NavItem[] {
+          text: 'Mintable Assets',
+          collapsed: true,
+          items: [
+            {
+              text: 'Mint Assets',
+            },
+            {
+              text: 'Mint More of a Mintable Asset',
+            },
+            {
+              text: 'Burn Assets',
+            },
+          ],
+        },
+        {
+          text: 'Non-Mintable Assets',
+          collapsed: true,
+          items: [
+            {
+              text: 'Work with Non-Mintable Assets',
+            },
+          ],
+        },
+        {
+          text: 'Tokens',
+          collapsed: true,
+          items: [
+            {
+              text: 'Create Asset-backed Tokens',
+            },
+            {
+              text: 'Create Non-Fungible Tokens (NFTs)',
+            },
+          ],
+        },
+      ],
+    },
+    {
+      text: 'Domains',
+      collapsed: true,
+      items: [
+        {
+          text: 'Register Domains',
+        },
+        {
+          text: 'Unregister Domains'
+        },
+        {
+          text: 'Update Domain Metadata',
+        },
+        // ... 
+        // access management for domains?
+      ],
+    },
+    {
+      text: 'Events and Filters',
+      collapsed: true,
+      items: [
+        {
+          text: 'Pipeline Events',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Data Events',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Trigger Events',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Advanced Filtering',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Block Stream',
+          collapsed: true,
+          items: [
+            {
+              text: 'Subscribe to Block Stream',
+            },
+            {
+              text: 'View output',
+            },
+          ],
+        },
+      ],
+    },
+    {
+      text: 'Instructions',
+      collapsed: true,
+      items: [
+        {
+          text: 'Use Instructions',
+        },
+        {
+          text: 'Combine Instructions via Expressions',
+        },
+      ],
+    },
+    {
+      text: 'Metadata',
+      collapsed: true,
+      items: [
+        {
+          text: 'Set Key Value',
+        },
+        {
+          text: 'Remove Key Value',
+        },
+        {
+          text: 'Access Metadata',
+        },
+      ],
+    },
+    {
+      text: 'Peers',
+      collapsed: true,
+      items: [
+        {
+          text: 'Register Peers',
+        },
+        {
+          text: "Check Current Peer's Load",
+        },
+        {
+          text: 'Find the Leader Among Running Peers',
+        },
+      ],
+    },
+    {
+      text: 'Queries',
+      collapsed: true,
+      items: [
+        {text: 'Use queries'},
+        {text: 'Filter query results'},
+        {text: 'Use Lazy Pagination'},
+        {text: 'Query Connected Peers'},
```

Comment:
```suggestion
        {text: 'Use queries'},
        {text: 'Filter query results'},
        {text: 'Use Sorting and Pagination'},
        {text: 'Query Connected Peers'},
```

there is no such thing as lazy pagination, there is server-side cursor which is active for every query and is opaque to the user. Server-side cursor doesn't require it's own section but one of the examples should contain example of how to set a `fetch-size`

Reply:
how about the client side cursor handling while iterating over the 'pages'? totally encapsulated in an SDK API?

Reply:
cursor handling is encapsulated in the SDK API and not visible to the user